apps/web/.env.test

```
@@ -1,11 +1,13 @@
 NEXT_PUBLIC_BASE_URL=http://localhost:3002
 AUTH_URL=$NEXT_PUBLIC_BASE_URL
 SECRET_PASSWORD=abcdef1234567890abcdef1234567890
 DATABASE_URL=postgres://postgres:postgres@localhost:5450/rallly
 SUPPORT_EMAIL=support@rallly.co
-SMTP_HOST=0.0.0.0
+SMTP_HOST=localhost
 SMTP_PORT=1025
 QUICK_CREATE_ENABLED=true
 CRON_SECRET=1234567890abcdef1234567890abcdef1234
 INITIAL_ADMIN_EMAIL=initial.admin@rallly.co
+SMTP_REJECT_UNAUTHORIZED=false
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)